from0k2bp.tex

```diff
@@ -322,18 +331,18 @@
 example, take the encoding of the point $G$, in binary, perhaps compressed
 or uncompressed form, take the SHA256 of that binary string and treat it
 as the $x$-coordinate of an elliptic curve point (assuming a curve of order
-$\simeq 2^256$). Not all 256 bit hash digests will be such $x$-coordinates, but about
+$\simeq 2^{256}$). Not all 256 bit hash digests will be such $x$-coordinates, but about
 half of them are, so you can just use some simple iterative algorithm
-(e.g. append byte ``1'', ``2''\ldots{} after the encoding of $G$) to create
-not just one such NUMS points , but a large set of them like $H_1, H_2,\ldots ,H_n$. And
+(e.g. append byte ``1'', ``2'', \ldots{} after the encoding of $G$) to create
+not just one such NUMS point $H$, but a large set of them like $H_1, H_2,\ldots, H_n$. And
 indeed we will make heavy use of such ``sets of pre-determined curve
 points for which no one knows the relative discrete logs'' later.
 
-Assuming has been thus constructed, let's think about whether the
+Assuming they have been thus constructed, let's think about whether the
 \textbf{binding} property of the commitment holds true:
 
-If I made a commitment to a single value in the form $C=rH + aG$, and I later am
-able to find two scalar values $s, b$ such that $C=sH+bG$, it \emph{proves }that
+If I made a commitment to a single value in the form $C=rH + aG$, and I'm later
```

Comment:
"I'm later able" is not better than "I later am able"

Reply:
Reverted